bindings/go/input/dir/blob_test.go

```diff
@@ -127,14 +129,16 @@ func TestGetV1DirBlob_Reproducibility(t *testing.T) {
 			require.NoError(t, err)
 
 			// Create new blob.
-			b, err = dir.GetV1DirBlob(ctx, dirSpec)
+			b, err = dir.GetV1DirBlob(ctx, dirSpec, tempDir)
 			require.NoError(t, err)
 			require.NotNil(t, b)
 
 			// Read the tar data after modification.
 			readerAfter, err := b.ReadCloser()
 			require.NoError(t, err)
-			defer readerAfter.Close()
+			defer func(readerBefore io.ReadCloser) {
+				require.NoError(t, readerBefore.Close())
+			}(readerAfter)
```

Comment:
```suggestion
			defer func() {
				require.NoError(t, readerAfter.Close())
			}()
```

Parameter name seems confusing. But generally, I think you can just use a closure here.